src/app/components/countdown/Countdown.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useRef, useEffect } from 'react'
+import FlipNumbers from 'react-flip-numbers'
+import './Countdown.scss'
+import tan from '../../../globals/zothacks-styles.scss'
+
+const Countdown = (props) => {
+
+    const ref = useRef(null);
+
+    const [timer, setTimer] = useState('00:00:00:00');
```

Comment:
Nitpick: I am not entirely fond of this format as some screen readers interpret it as hours, minutes, seconds, followed by extra numbers rather than days, hours, minutes seconds, but this is a bit beyond the scope of your changes.